Managers/PrimitiveMan.cpp

```diff
@@ -4,6 +4,24 @@
 
 namespace RTE {
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void PrimitiveMan::DrawLinePrimitive(int player, const Vector &startPos, const Vector &endPos, unsigned char color, int thickness) {
+		if (thickness > 1) {
+			// Cheese thick line drawing with opposing right-angle triangles.
```

Comment:
Technically should use SceneMan.SceneWrapsX instead of true in the shortest distance call

Also, I don't think we need the comment here - it doesn't really add anything. The code makes it clear that triangles are being drawn, and it's obvious that that must be used to draw the line.